torchvision/transforms/functional.py

```diff
@@ -1461,3 +1461,83 @@ def equalize(img: Tensor) -> Tensor:
         return F_pil.equalize(img)
 
     return F_t.equalize(img)
+
+
+def elastic_transform(
+    img: Tensor,
+    alpha: List[float],
+    sigma: List[float],
+    interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    fill: Optional[List[float]] = None,
+    displacement: Optional[Tensor] = None,
```

Comment:
In terms of API, there are two things I'm a bit unhappy:
- if user provides `displacement` and `alpha` and `sigma` then only `displacement` is taken into account =>
either we have make both optional and warn/raise an error that only one is possible.
- if we are using PIL image as input, how user should define a displacement. Mixing PIL image as input and `displacement` as tensor does not look as a proper solution...

Reply:
1) sounds reasonable to do it that way.
2) okay, so we also accept np.array as the displacement type and wrap it with torch.tensor internally?

Reply:
> okay, so we also accept np.array as the displacement type and wrap it with torch.tensor internally?

Yes, that's can be an option

Reply:
Okay, I implemented both changes and pushed it.

Reply:
> warn/raise an error that only one is possible.

@JLrumberger we need explicitly check that user does not provide both alph/sigma and displacement. if displacement is given, both alpha and sigma should be None, otherwise raise an error.


As for displacement, JIT can not work with numpy input => if input is tensor, then displacement should be a tensor, if input is PIL.Image, displacement should be a ndarray.